androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/screens/journal/triage/previousCare/PreviousCareScreen.kt

```diff
@@ -0,0 +1,36 @@
+package com.EENX15_22_17.digital_journal.android.screens.journal.triage.previousCare
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.EENX15_22_17.digital_journal.android.R
+import com.EENX15_22_17.digital_journal.android.ui.DetailPageWrapper
+import com.EENX15_22_17.digital_journal.android.ui.components.TitledSectionTemp
+import com.EENX15_22_17.digital_journal.android.ui.theme.Colors
+import se.predicare.journal.screens.PreviousCareDto
+
+@Composable
+fun PreviousCareScreen(
+    journalId: String,
+    onBackClicked: () -> Unit,
+    onMenuClicked: () -> Unit
+) {
+    DetailPageWrapper(
+        title = stringResource(id = R.string.previousCare),
+        titleColor = Colors.triagePrimary,
+        onBackClicked = onBackClicked,
+        onMenuClicked = onMenuClicked
+    ) {
+        Column(modifier = Modifier.padding(top=20.dp)) {
+            TitledSectionTemp(title = stringResource(id = R.string.previousHealtCare)) {
```

Comment:
Why do we have two types of titled section? we should merge them, or if they have different purposes, rename them.